app/assets/stylesheets/snowcrash/base.scss

```diff
@@ -289,6 +289,11 @@ body {
             font-size: 10px;
             padding: 0 0 1em 3em;
           }
+          &.node {
```

Comment:
Two comments:

- Maybe there is no need to apply this change to the `node` class only, it can be applied above, like here: https://github.com/dradis/dradis-ce/blob/master/app/assets/stylesheets/snowcrash/base.scss#L170

- we may use `break-word` instead of `break-all`, so if the name have a white space it will break there. 

Thank you for your contribution!

Reply:
hi @dormi in the original issue (#228) it is a word without space . so if we give `break-word` it will not serve that purpose. node names can actually be without space.